scripts/liminal

```diff
@@ -113,7 +124,7 @@ def start():
         # initialize liminal home by default
         environment.get_liminal_home()
         docker_compose_path, project_dir = get_docker_compose_paths()
-        docker_compose_command('up', [])
+        docker_compose_command('up', ['-d'])
```

Comment:
-d means users don't by default see the logs.
it also enables the "stop" command to become available.
there is a question on how users see logs esp. errors.
I don't think it's ok to expect them to do docker ps and docker logs- some of them don't know we're running docker.

I suggest to add liminal logs command which delegates to docker compose logs. including the ability for -f and tail (delegate directly to docker compose). 
Also need to update the README with the stop and logs command, ideally also add screenshots of outptus users should expect

Reply:
I added liminal logs --follow/--tail.
As well, I updated the README. Let me know if screenshots are still required.